geomagio/JSON/JSONWriter.py

```diff
@@ -0,0 +1,212 @@
+from __future__ import absolute_import
+from builtins import range
+
+from io import BytesIO
+from collections import OrderedDict
+from datetime import datetime
+import json
+import numpy
+import textwrap
+from .. import ChannelConverter, TimeseriesUtility
+from ..edge import EdgeFactory
+from ..TimeseriesFactoryException import TimeseriesFactoryException
+from ..Util import create_empty_trace
+
+
+class JSONWriter(object):
+    """JSON writer.
+    """
+
+    def __init__(self):
+        self.dictionary = OrderedDict()
+
+    def write(self, out, timeseries, channels, **kwargs):
+        """write timeseries to json file
+
+        Parameters
+        ----------
+        out: file object
+            file object to be written to. could be stdout
+        timeseries: obspy.core.stream
+            timeseries object with data to be written
+        channels: array_like
+            channels to be written from timeseries object
+        """
+        request = kwargs.get('request')
```

Comment:
Add a `url` parameter that defaults to None, the JSON format could be used outside a web service context.